src/main/java/org/graphqlcrudjava/types/TypeMapImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.graphqlcrudjava.types;
+
+import java.sql.Types;
+
+public class TypeMapImpl implements TypeMap {
```

Comment:
Missing database info. What dB we have here

Reply:
Which all Db are to be provided?

Reply:
I mean this specific type mappings are from Teiid or are coming from PG. Different databases will have different types so worth to name mapper or something to see what we have here

Reply:
JDBC driver actually already masks the identity of the database type underneath, the driver is responsible for mapping its types into the JDBC types, so as long as we are consistent with JDBC types we will match across the DBs